py/server/deephaven/updateby.py

```diff
@@ -48,6 +49,15 @@ class BadDataBehavior(Enum):
     POISON = _JBadDataBehavior.POISON
     """Allow the bad data to poison the result. This is only valid for use with NaN"""
 
+class DeltaControl(Enum):
+    """An Enum defining ways to handle null values during update-by Delta operations."""
```

Comment:
This should include some clarification of what "Delta operations" are, since without that, a new user will not have any idea what this is and if they have arrived at the right docs to help them.